packages/web-components/src/presence-badge/svg/svg.ts

```diff
@@ -0,0 +1,38 @@
+// © Microsoft Corporation. All rights reserved.
+
+// FILLED ICONS:
```

Comment:
Now that we have the fluent-system-icons published as svg, we should directly import those.

Reply:
I still suggest what I said above, but also now that we have the `options` pattern we should allow the icons to be overridden as options when the component is composed instead of as slots. That would create a more customizable and reusable experience instead of overriding everywhere you might use a presence badge within an app.